lib/model/folder.go

```diff
@@ -8,12 +8,17 @@ package model
 
 import (
 	"context"
+	"errors"
 	"time"
 
 	"github.com/syncthing/syncthing/lib/config"
+	"github.com/syncthing/syncthing/lib/ignore"
+	"github.com/syncthing/syncthing/lib/sync"
 	"github.com/syncthing/syncthing/lib/watchaggregator"
 )
 
+var errWatchNotStarted error = errors.New("Filesystem change watcher has not been started yet")
```

Comment:
The error itself should just be a sentence fragment without upper case, with whatever does the actual logging in the end providing the context.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

`errors.New("not started")`

Reply:
That comes up in almost every review - I seem to be unable to fix it in my ... brain. I just wanted to add `grep -rP '(errors.New|fmt.Errorf)\("[A-Z]' lib/ cmd/` as a crutch to my local "build script", and there's like a lot of pre-existing capitalized error strings (not from me though xD ).